api-model/src/main/java/io/enmasse/address/model/AddressList.java

```diff
@@ -12,6 +12,9 @@
  * Type for address lists.
  */
 public class AddressList extends ArrayList<Address> {
+
+    private static final long serialVersionUID = 1L;
+
```

Comment:
Do we actually care about serialization (i.e. we are going to modify this if we change the class in a non-compatible way) or are we just trying to suppress warnings (in which case a @SuppressWarnings annotation might express the intent better)

Reply:
Yes this all about suppressing warnings. The latter case would work fine as well. Whatever you prefer.

Reply:
Or you could just turn off the warnings in Eclipse :-)

I generally hate the serialVersionUID being included because it looks like an intention for the class to be serialised, and that an attempt will be made to either maintain compatibility or modify the Uid when necessary.  If the intent is to suppress warnings I'd probably use the appropriate annotation (yes there are warnings but I just don't care) or simply turn them off in your IDE)